ChangeLog

```diff
@@ -61,6 +61,10 @@ Features
    * Add MBEDTLS_REMOVE_3DES_CIPHERSUITES to allow removing 3DES ciphersuites
      from the default list (enabled by default). See
      https://sweet32.info/SWEET32_CCS16.pdf.
+   * Add support for parsing subject alternative name, of type otherName,
+     HardwareModuleName, as defined in rfc 4108 section 5.
+   * Add support for parsing certificate policies extension, as defined in
+     rfc 5280 section 4.2.1.4. Currently only supports "Any Policy" policy.
```

Comment:
Minor: Write full sentences, e.g. `Currently, only the "Any Policy" policy is supported.`